src/components/ImportForm/ApplicationSection/ApplicationSection.tsx

```diff
@@ -1,21 +1,19 @@
 import * as React from 'react';
 import { TextContent, Text, HelperText, FormSection } from '@patternfly/react-core';
 import { InputField } from '../../../shared';
+import { useValidApplicationName } from '../utils/useValidApplicationName';
 
 const ApplicationSection: React.FunctionComponent = () => {
+  const [validName] = useValidApplicationName();
```

Comment:
Do we actually need to do a validation for just placeholder text? We are already doing that on next.

Reply:
The hook returns an incremented name that isn't used, like `my-first-app-$n`. UX wanted the placeholder value to increment https://issues.redhat.com/browse/HAC-2353

Reply:
How does it adds value if this is just a placeholder text? It just unnecessarily slows the process for the user. It would have made sense when we were actually recommending the application in the input field and not just showing a placeholder text which eventually needs to be entered by the user. @beaumorley What do you think?

Reply:
@beaumorley ^^

Reply:
@rohitkrai03 @beaumorley The placeholder text seems weird to me. Can we automatically populate the field with "my-first-app" (reg text not placeholder) and the placeholder "Enter name" only displays if the user removes all the text? So if user comes here they can simply hit enter.

Reply:
@Misjohns updated!

https://user-images.githubusercontent.com/20013884/214018741-57b9b201-0caf-4f6c-ab81-6fdf73d67505.mp4

Reply:
@rottencandy Do we need the `onNext` validation of application name anymore? i think we can remove that check if the application name is always recommended based on what can be created.

Reply:
But we still allow users to change to a different name, should we not do a check on the field in that case?

Reply:
Well, yeah in that case we still need that check.